extensions/SampleExtension/SettingsProvider/SettingsUIController.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using Microsoft.Windows.DevHome.SDK;
+using SampleExtension.Helpers;
+using Serilog;
+using Windows.Foundation;
+
+namespace SampleExtension.Providers;
+
+internal sealed class SettingsUIController() : IExtensionAdaptiveCardSession
+{
+    private static readonly Lazy<ILogger> _logger = new(() => Serilog.Log.ForContext("SourceContext", nameof(SettingsUIController)));
```

Comment:
Not for this change: can we add a small doc about logging? (Depending on how much we want to document, maybe to https://github.com/microsoft/devhome/tree/main/docs/tools, maybe inside https://github.com/microsoft/devhome/blob/main/docs/tools/advanced-tool-behavior.md?)
Not sure we need to go into setting up the logger for a new process (although we could, in the extension documentation) but using logging inside "regular" Dev Home and how/when to use the Lazy logger could be useful.

Reply:
This is a good idea to add to extension documentation on how Dev Home does logging and encouraging extension developers to follow a similar pattern (and why we have the patterns we do).